base/applications/cmdutils/timeout/lang/ru-RU.rc

```diff
@@ -0,0 +1,33 @@
+LANGUAGE LANG_RUSSIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "ReactOS Timeout Utility\n\
+\n\
+TIMEOUT [/?] [/T] таймаут [/NOBREAK]\n\
+\n\
+Описание:\n\
+    Команда позволяет организовать режим ожидания фиксированного интервала времени,\n\
+    в секундах, либо интервала до нажатия клавиши на клавиатуре.\n\
+\n\
+Синтаксис:\n\
+    /?        Показать эту справку.\n\
+\n\
+    /T delay  Время ожидания в секундах. Допустимый интервал: от -1 до 99999 секунд.\n\
+              Значение таймаута, равное -1 задает неограниченное время ожидания \n\
+              до нажатия клавиши.\n\
+              Ключ ""/T"" не является обязательным. Допускается указание значения\n\
+              таймера без указания ключа.\n\
+\n\
+    /NOBREAK  Игнорировать нажатия клавиш, кроме сочетания Ctrl+C.\n\
+"
+    IDS_ERROR_OUT_OF_RANGE "Ошибка: Значение таймера должно находиться в пределах диапазона (от -1 до 99999).\n"
+    IDS_ERROR_INVALID_HANDLE_VALUE "Ошибка: Не удалось получить стандартный дескриптор консоли (ошибка %lu).\n"
+    IDS_ERROR_READ_INPUT "Ошибка: Невозможно прочитать консольный ввод (ошибка %lu).\n"
+    IDS_ERROR_NO_TIMER_VALUE "Ошибка: Значение таймера должно быть указано (от -1 до 99999).\n"
+    IDS_ERROR_ONE_TIME "Ошибка: Требуется только одно значение таймера.\n"
+    IDS_NOBREAK_INPUT "Нажмите Ctrl + C для выхода..."
```

Comment:
Use "Ctrl+C" without spaces for consistency with the other line (IDS_NOBREAK_INPUT_COUNT).

Reply:
Fixed.